recipes/forestplot/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "forestplot" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d711f46bd25ad66287763a6d7a795f89acb43e7306feda906d79003563748134
+
+build:
+  number: 0
+  noarch: python  
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python >=3.6
+
+test:
+  imports:
+    - forestplot
+
+about:
+  home: "https://github.com/lsys/forestplot"
+  license: MIT
+  license_family: MIT
+  license_file: https://github.com/LSYS/forestplot/blob/main/LICENSE
```

Comment:
conda-build won't download that for you. You'll need to include that file here in with your meta.yaml.

Reply:
thanks!